packages/eas-cli/src/commands/metadata/lint.ts

```diff
@@ -0,0 +1,79 @@
+import { Platform } from '@expo/eas-build-job';
+import { EasJsonAccessor } from '@expo/eas-json';
+import { Flags } from '@oclif/core';
+
+import EasCommand from '../../commandUtils/EasCommand';
+import Log from '../../log';
+import { loadConfigAsync } from '../../metadata/config/resolve';
+import { MetadataValidationError, logMetadataValidationError } from '../../metadata/errors';
+import { enableJsonOutput, printJsonOnlyOutput } from '../../utils/json';
+import { getProfilesAsync } from '../../utils/profiles';
+
+export default class MetadataLint extends EasCommand {
+  static override description = 'validate the local store configuration';
+
+  static override flags = {
+    json: Flags.boolean({
+      description: 'Enable JSON output, non-JSON messages will be printed to stderr',
+      default: false,
+    }),
+    profile: Flags.string({
+      description:
+        'Name of the submit profile from eas.json. Defaults to "production" if defined in eas.json.',
+    }),
+  };
+
+  static override contextDefinition = {
+    // The metadata lint command is created to integrate in other dev tooling, like vscode-expo.
+    // These integrations might spam this command, so we avoid communicating with our services here.
+    // Note that this is an exception and you should normally use `ProjectConfig` instead.
```

Comment:
I think it reflected on using `ProjectDir` instead of `ProjectConfig`, right? Can remove it, but I believe this was put to discourage using `ProjectDir` directly.